pkg/machine/qemu/options_darwin_arm64.go

```diff
@@ -34,3 +35,17 @@ func (v *MachineVM) archRemovalFiles() []string {
 func getOvmfDir(imagePath, vmName string) string {
 	return filepath.Join(filepath.Dir(imagePath), vmName+"_ovmf_vars.fd")
 }
+
+func getEdk2CodeFd(name string) string {
```

Comment:
Can you throw a comment on this as to why it's necessary, for anyone looking at the code in the future?